libpod/runtime.go

```diff
@@ -486,14 +486,17 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (retErr error) {
 			}
 
 		}
-		// If the file doesn't exist, we need to refresh the state
+		// If the file doesn't exist, and we are not running inside of
+		// a container we need to refresh the state
 		// This will trigger on first use as well, but refreshing an
 		// empty state only creates a single file
 		// As such, it's not really a performance concern
-		if os.IsNotExist(err) {
-			doRefresh = true
-		} else {
-			return errors.Wrapf(err, "error reading runtime status file %s", runtimeAliveFile)
+		if os.Getenv("container") == "" {
```

Comment:
We're just trading one set of issues for another here - if you actually *are* the first Podman call after a reboot, things are going to be massively messed up.

Reply:
The first podman would have been the outer shell which would have done the reset.  I think the other cases are unlikely.  Podman running within Docker, CRI-O Buildah as the first time.